cmd/auth/main.go

```diff
@@ -22,7 +24,10 @@ import (
 )
 
 type configuration struct {
-	DevMode bool `envconfig:"DEV_MODE"`
+	DevMode           bool   `envconfig:"DEV_MODE"`
+	APIKey            string `envconfig:"API_KEY"`
+	MasterKey         string `envconfig:"MASTER_KEY"`
+	CustomTokenSecret string `envconfig:"CUSTOM_TOKEN_SECRET"`
```

Comment:
In next, api key, master key and custom token is provided by the tenant config which is different per request. After discussion, we support provide the tenant config when creating logger and the formatter should be set there.